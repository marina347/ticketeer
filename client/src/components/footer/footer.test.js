import React from "react";
import { shallow } from "enzyme";
import "jest-styled-components";
import Footer from "./footer.component";

describe("Footer component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });

  it("Should render Footer component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
