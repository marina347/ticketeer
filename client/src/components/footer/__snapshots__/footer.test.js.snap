// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should render Footer component 1`] = `
<div
  className="footer"
>
  <p
    className="footer__copyright"
  >
    Built by Marina Civcija. Copyright Â© by Marina Civcija.
  </p>
</div>
`;
